### ISSUE TITLE
Prepare datasets and annotations

### ISSUE THREAD

**Issue by jer1ry** [PeizeSun/TransTrack#79]
When I was doing Prepare datasets and annotations, there was an error using cp -r /path_to_crowdhuman_dataset/CrowdHuman_train crowdhuman/CrowdHuman_train.
Error: cp: cannot stat '/path_to_crowdhuman_dataset/CrowdHuman_train': No such file or directory

**Comment by tran7777777** [comment]
I created the crowdhuman_train and mot_train folders directly by hand, and put the training data sets in mot and crowdhuman. I don't know if it's right, because the accuracy dropped after I ran it.

**Comment by jer1ry** [comment]
@tran7777777 
Hi
Are you using multiple GPUs or single GPU? I am currently unable to run the code successfully

**Comment by jer1ry** [comment]
@tran7777777 
Can you tell me the cuda version, python version, torch version used? Thanks

**Comment by tran7777777** [comment]
11.3  3.8  1.12.1
single GPU

**Comment by jer1ry** [comment]
@tran7777777 
thank you!I think we can communicate more. can you give me a contact.

**Comment by z972778371** [comment]
> I created the crowdhuman_train and mot_train folders directly by hand, and put the training data sets in mot and crowdhuman. I don't know if it's right, because the accuracy dropped after I ran it.
Hi, could you tell me the structure under the crowdhuman folder? After downloading the CH dataset, I find there are three folders called CrowdHuman_train01, CrowdHuman_train02 and CrowdHuman_train03. Should I put them in the crowdhuman folder or mix the images in the 3 folders into one folder named CrowdHuman_train?
